packages/vendure-plugin-invoices/src/api/invoice.service.ts

```diff
@@ -281,45 +295,72 @@ export class InvoiceService implements OnModuleInit, OnApplicationBootstrap {
   }
 
   async downloadMultiple(
-    channelId: string,
+    ctx: RequestContext,
+    // channelId: string,
     invoiceNumbers: string[],
     res: Response
   ): Promise<ReadStream> {
     const nrSelectors = invoiceNumbers.map((i) => ({
       invoiceNumber: i,
-      channelId,
+      channelId: ctx.channelId,
     }));
-    const invoices = await this.invoiceRepo.find({
-      where: nrSelectors,
+    const invoiceRepo = this.connection.getRepository(ctx, InvoiceEntity);
+    const invoices = await invoiceRepo.find({
+      where: {
+        channelId: In(nrSelectors.map((i) => i.channelId)),
+        invoiceNumber: In(nrSelectors.map((i) => i.invoiceNumber)),
+      },
     });
     if (!invoices) {
       throw Error(
-        `No invoices found for channel ${channelId} and invoiceNumbers ${invoiceNumbers}`
+        `No invoices found for channel ${ctx.channelId} and invoiceNumbers ${invoiceNumbers}`
       );
     }
     return this.config.storageStrategy.streamMultiple(invoices, res);
   }
 
   async upsertConfig(
-    channelId: string,
-    input: InvoiceConfigInput
+    input: InvoiceConfigInput,
+    ctx?: RequestContext,
+    channelToken?: string
```

Comment:
Lets remove this, it's only used in the dev-server I believe? We should just create a CTX in the dev-server then.

```suggestion
    input: InvoiceConfigInput,
    ctx: RequestContext
```